release-notes/changes-in-timescaledb-2.md

```diff
@@ -320,6 +320,40 @@ After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due
 continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the 
 desired interplay. 
 
+Another change in 2.0 is that `drop_chunks` and the retention policy do not refresh continuous 
+aggregates on the dropped chunks. Previously dropping a chunk would call a refresh on each 
+continuous aggregate to recompute it according to existing invalidations. The refresh was 
+unlikely to include all updates due to `ignore_invalidations_older_than`, which prevents to 
+record invalidations for updates older than the threshold. If a user carefully crafted 
+parameters to the retention policy and the continuous aggregate that no update is missed, 
+the refresh could include previously dropped values leading to unexpected outliers in 
+aggregated values. TimescaleDB 2.0 introduces new function 
+`_timescaledb_internal.refresh_continuous_aggregate`, which refreshes 
+exactly a chunk, and can be used together with drop_chunks. For example, following action can 
+be defined:
```

Comment:
I think this is a little bit too much "in the weeds". Suggested simplification:

```suggestion
Another change in 2.0 is that `drop_chunks` and the retention policy will no longer automatically refresh continuous aggregates to account for backfill in chunks when they are dropped. Previously, the goal was to ensure that all backfill in a chunk was processed prior to dropping it. However, this didn't always work as intended (e.g., if the materialization was far behind), and it also made retention policies slow.

If the old behavior of processing a chunk's backfill during drops is still desired, 2.0 offers the experimental function `_timescaledb_internal.refresh_continuous_aggregate`, which refreshes exactly one chunk. The function can be used together with `drop_chunks` in a single transaction, making it possible to define the following user-defined action:

```

Reply:
The suggestion changes the message I was out for. Also the backfill is not the only case. I will try to make my formulation shorter with keeping in mind your suggestion.

Reply:
I like this change, @erimatnor.  Because this is experimental/internal, I assume that's why it's not in the docs.  Should we suggest here what the args are?  Does it take in (chunk, start_offset, end_offset) ?

Reply:
@mfreed I will put the function signature in a separate line, so arguments fit also. I also added an example of a user policy using it. I hope the example will be reduced for the final release as it demonstrates a hole in the information view.

Reply:
As always, this is a suggestion. The mention of only backfill can easily be modified. My main concern remains: that the original paragraph is too much in the weeds, which gets in the way of the important message that there's a new alternative.

Reply:
Thank you for the suggestion, I updated the formulation.